cache/restore.go

```diff
@@ -140,6 +142,17 @@ func (r *restorer) createConfig(input RestoreCacheInput) (restoreCacheConfig, er
 		return restoreCacheConfig{}, fmt.Errorf("the secret 'BITRISEIO_BITRISE_SERVICES_ACCESS_TOKEN' is not defined")
 	}
 
+	maxConcurrency := uint(0)
+	maxConcurrencyStr := r.envRepo.Get("BITRISEIO_DEPENDENCY_CACHE_MAX_CONCURRENCY")
```

Comment:
is this now the standard that we prefix these env vars with `DEPENDENCY_CACHE`?

Reply:
not officially, yet